app/views/claims/support/schools/new.html.erb

```diff
@@ -10,7 +10,7 @@
 
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
-        <div class="govuk-form-group">
+        <div class="govuk-form-group<%= " govuk-form-group--error" if f.object.errors.present? %>">
```

Comment:
Using `class_names` might clean this up. The inline `if` only applies on the additional class, but it might not be immediately noticeable.

```suggestion
        <%= content_tag(:div, class: class_names("govuk-form-group", "govuk-form-group--error": f.object.errors.present? %>
```

Not a blocker though!

Reply:
Swapped out in [4cd376b](https://github.com/DFE-Digital/itt-mentor-services/pull/137/commits/4cd376bb73b90b6679cf4d39907d07f34a2ef49c)